eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -30,15 +30,22 @@ import scala.concurrent.duration._
 import scala.concurrent.{Await, ExecutionContext, Future, Promise}
 
 /**
+  * Setup eclair from a datadir.
+  * <p>
   * Created by PM on 25/01/2016.
+  *
+  * @param datadir  directory where eclair-core will write/read its data
+  * @param overrideDefaults
+  * @param actorSystem
+  * @param seed_opt optional seed. If set, will be used by eclair, and also used as master by the embedded electrum watcher.
```

Comment:
> ~optional seed. If set, will be used by eclair, and also used as master by the embedded electrum watcher.~

> optional seed, if set eclair will use it instead of generating one and won't create a seed.dat file.